src/tree/ActionTreeItem.ts

```diff
@@ -56,6 +57,21 @@ export class ActionTreeItem extends AzureTreeItem implements IAzureResourceTreeI
         return this.data.event;
     }
 
+    public async loadMoreChildrenImpl(_clearCache: boolean, _context: IActionContext): Promise<AzExtTreeItem[]> {
+        const { owner, name } = getRepoFullname(this.parent.repositoryUrl);
+        const token: string = await getGitHubAccessToken();
+        const requestOption: gitHubWebResource = await createGitHubRequestOptions(token, `${githubApiEndpoint}/repos/${owner}/${name}/actions/runs/${this.data.id}/jobs`);
+        const githubResponse: IncomingMessage & { body: string } = await requestUtils.sendRequest(requestOption);
+        const gitHubJobs: { jobs: GitHubJob[] } = <{ jobs: GitHubJob[] }>JSON.parse(githubResponse.body);
+        return gitHubJobs.jobs.map((job => {
```

Comment:
Like Eric mentioned before, we should use the `createTreeItemsWithErrorHandling` especially since it's a custom call that we're making.